CHANGELOG.md

```diff
@@ -30,6 +30,7 @@ ENHANCEMENTS:
 * The test `run` outputs can now be used in the test `provider` blocks defined in test files. ([#2543](https://github.com/opentofu/opentofu/pull/2543))
 * Provider instance keys now automatically converted to string ([#2378](https://github.com/opentofu/opentofu/issues/2378))
 * Remove progress messages from commands using -concise argument ([#2549](https://github.com/opentofu/opentofu/issues/2549))
+* Upgrade aws-sdk version to include `mx-central-1` region. Also, `skip_s3_checksum=true` now blocks the [aws-sdk new default S3 integrity checks](https://github.com/aws/aws-sdk-go-v2/discussions/2960)([#2596](https://github.com/opentofu/opentofu/pull/2596))
```

Comment:
> Also, `skip_s3_checksum=true` now blocks the aws-sdk new default S3 integrity checks

I believe we should put here actual user-facing changes. What does it do from the user perspective?

Reply:
I wanted to highlight that the behaviour for skipping checksums suffered a change. Just in case somebody is stumbling into an issue with it. This way would be easier to find what changed.
From the user perspective, it is meant to work the same, nothing changes.
If we keep only the user facing things in here, let me remove that second statement.

Reply:
I believe we could mention it as an internal change. There is an example for 1.9 release: https://github.com/opentofu/opentofu/blob/v1.9/CHANGELOG.md (`INTERNAL CHANGES` block)